src/types/frozenset.rs

```diff
@@ -20,7 +20,7 @@ impl<'py> PyFrozenSetBuilder<'py> {
     /// panic when running out of memory.
     pub fn new(py: Python<'py>) -> PyResult<PyFrozenSetBuilder<'py>> {
         Ok(PyFrozenSetBuilder {
-            py_frozen_set: PyFrozenSet::empty(py)?,
+            py_frozen_set: PyFrozenSet::empty_bound(py)?,
```

Comment:
Oops I missed adding a deprecation on `PyFrozenSet::empty`! Potentially can you push that as a follow up please?